### PR TITLE
Grab Telemetry Uptake using main summary.

### PR DESCRIPTION
**Pull request by Natim**
- BETA: https://sql.telemetry.mozilla.org/queries/50461/source#table
- DEVEDITION: https://sql.telemetry.mozilla.org/queries/50463/source#table

### CODE REVIEW

**Comment by Natim** [comment]
For nightly because of the latency it will always be null, so I will deactivate this information for nightly.

**Comment by jcristau** [comment]
Can you please explain, in plain English, what you're trying to accomplish with this check?  I don't know that reading 10 different telemetry queries is going to help me understand what you're trying to get at, and I have no idea what is or isn't available in which obscure corner of the various telemetry frontends, so I'm kind of lost as to what feedback I'm asked for here.  Thanks!

**Comment by Natim** [comment]
> Can you please explain, in plain English, what you're trying to accomplish with this check

I am trying to get you the percentage of users using the new version with regards to all the version in use in a given channel.
